dominique-mueller/devdom.io
# chore(dependencies): Upgrade dependencies, update CI

- Update npm dependencies
- Update Hugo to latest version
- Remove pygments (no longer necessay as replaced by Chroma, which is included in Hugo now)
- Add Hugo download script
- Cleanup & refactor Travis CI configuration file
- Fix Gulp tasks
